legend-engine-xt-nonrelationalStore-mongodb-javaPlatformBinding-pure/src/main/resources/core_nonrelational_mongodb_java_platform_binding/test/pureSetup.txt

```diff
@@ -123,7 +123,7 @@ MongoDBConnection test::mongodb::testConnection
 {
     database: userDatabase;
     store: test::mongodb::mydatabase;
-    serverURLs: [localhost:49829];
+    serverURLs: [localhost:57482];
```

Comment:
we can probablystop committing this file :) or fix it to 27017 the default mongo port 27017 - ideally you would want the container to map that port)